app/src/main/java/protect/card_locker/LoyaltyCardField.java

```diff
@@ -11,6 +11,7 @@ public enum LoyaltyCardField {
     cardId,
     barcodeId,
     barcodeType,
+    usage_number,
```

Comment:
Please use camel-case like everywhere else. And please keep the same order as in the database class (so, below archiveStatus)